journal/modernized/32-epistles.adoc

```diff
@@ -546,6 +569,7 @@ that so it may be seen we are in the footsteps of those who were given up in str
 understanding, time and substance,
 to spend and be spent in the service of the Lord and his people.
 
+[.signed-section-signature]
 William Edmundson.
 
 === Concerning Offerings to God, in Prayer and Supplication.
```

Comment:
This is really unusual (I keep saying that!)... because it doesn't say something like "William Edmundson to blah blah".... but partly because it describes what is contained in this letter, and partly to be consistent with the other letter headings in the chapter, I would label this with a [.letter-heading]